module-2/task1/fibonacci/fibonacci.go

```diff
@@ -0,0 +1,48 @@
+package fibonacci
+
+import (
+    "errors"
+)
+
+func Fibonacci(n interface{}) interface{} {
+    switch n.(type) {
```

Comment:
you can change this to d := n.(type) and go automatically cast d to the proper type  in each case,

Reply:
Thanks! Fixed